include/sharg/enumeration_names.hpp

```diff
@@ -212,9 +212,10 @@ namespace sharg
  * \experimentalapi{Experimental since version 1.0.}
  */
 // clang-format off
+// Doxygen 1.9.5 bug: https://github.com/doxygen/doxygen/issues/9552
```

Comment:
```suggestion
// `SHARG_DOXYGEN_ONLY((size_t value))` is needed for Doxygen 1.9.5 
// Doxygen 1.9.5 bug: https://github.com/doxygen/doxygen/issues/9552
```

Otherwise it's hard to know later on if the brief above or something below is buggy.